### PR TITLE
Insert Nano V1.3 board to select in BlueR Gcode

### PR DESCRIPTION
**Pull request by ils15**
Some BlueR arrived with MKS Robin Nano V 1.3

This adjust alow to user quick select thus board by uncomment file

### CODE REVIEW

File: config/examples/Two Trees/BlueR/BlueR V3/Configuration.h [hunk 1]
```diff
@@ -20,11 +20,10 @@
  *
  */
 #pragma once
-#error "Don't build with import-2.1.x configurations!"
-#error "Use the 'bugfix...' or 'release...' configurations matching your Marlin version."
```
Comment (thisiskeithb) [inline config/examples/Two Trees/BlueR/BlueR V3/Configuration.h]: These lines should not be removed. Please revert this change.

**Comment by Joeben349** [comment]
Cashapp login issues
